### PR TITLE
#1871 Create a common helper function for Comment[Create|Delete]

### PR DESCRIPTION
**Pull request by batradhruv**
## The basics

- [ x] I branched from develop
- [x ] My pull request is against develop
- [ x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

https://github.com/google/blockly/issues/1871

### Proposed Changes

This PR will create a common Helper function within lockly.Events.CommentCreateDeleteHelper. It will be called by Blockly.Events.CommentCreate.prototype and Blockly.Events.CommentDelete.prototype

### Reason for Changes
Cleanup of code base. Reduce redundant code

### Test Coverage

I havent added new test cases. I would expect existing ones to pass successfully 

Tested on:

### CODE REVIEW

**Comment by batradhruv** [comment]
I signed it!

**Comment by batradhruv** [comment]
@rachel-fenichel Could you take a look at this PR please?

File: core/ws_comment_events.js [hunk 1]
```diff
@@ -215,6 +215,14 @@ Blockly.Events.CommentCreate.prototype.fromJson = function(json) {
...
  */
 Blockly.Events.CommentCreate.prototype.run = function(forward) {
+  Blockly.Events.CommentCreateDeleteHelper(forward);
+};
+
+/**
+ * Helper function for Comment[Create|Delete].prototype.run
+ * @param {boolean} forward True if run forward, false if run backward (undo).
+ */
+Blockly.Events.CommentCreateDeleteHelper = function(forward){
```
Comment (AnmAtAnm) [inline core/ws_comment_events.js]: Nit: Space between `)` and `{`

File: core/ws_comment_events.js [hunk 2]
```diff
@@ -215,6 +215,14 @@ Blockly.Events.CommentCreate.prototype.fromJson = function(json) {
  * @param {boolean} forward True if run forward, false if run backward (undo).
  */
 Blockly.Events.CommentCreate.prototype.run = function(forward) {
+  Blockly.Events.CommentCreateDeleteHelper(forward);
+};
+
+/**
+ * Helper function for Comment[Create|Delete].prototype.run
+ * @param {boolean} forward True if run forward, false if run backward (undo).
```
Comment (AnmAtAnm) [inline core/ws_comment_events.js]: I wonder if `create` or similar might be a better name than `forward` for the help that is used in both create and delete cases.

Comment (rachel-fenichel) [reply core/ws_comment_events.js]: +1

File: core/ws_comment_events.js (hunk 1 above)
Comment (batradhruv) [reply core/ws_comment_events.js]: Ack

File: core/ws_comment_events.js (hunk 2 above)
Comment (batradhruv) [reply core/ws_comment_events.js]: Sure. Ill change this to create

**Comment by batradhruv** [comment]
@AnmAtAnm have updated it var name to "create". Please take a look

**Pull request merged**